pkg/cmd/cli/backup/describe.go

```diff
@@ -118,5 +131,6 @@ func NewDescribeCommand(f client.Factory, use string) *cobra.Command {
 	c.Flags().BoolVar(&details, "details", details, "Display additional detail in the command output.")
 	c.Flags().BoolVar(&insecureSkipTLSVerify, "insecure-skip-tls-verify", insecureSkipTLSVerify, "If true, the object store's TLS certificate will not be checked for validity. This is insecure and susceptible to man-in-the-middle attacks. Not recommended for production.")
 	c.Flags().StringVar(&caCertFile, "cacert", caCertFile, "Path to a certificate bundle to use when verifying TLS connections.")
+	c.Flags().StringVar(&outputFormat, "o", outputFormat, "Output display format. Valid formats are 'json'. This flag only applies to a single backup")
```

Comment:
It's better to support both `-o` and `--output` to keep consistent with other commands.
Should we introduce a new format such as `plaintext` for the default output format? After reading the description of the flag, seems that it only supports one format JSON.

Reply:
Make sense.